fidelityfx-lib/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+use std::path::{Path, PathBuf};
+
+pub fn dll_path_vk() -> PathBuf {
```

Comment:
Just thought of this now: should we keep `windows` and the target architecture in the function name as well, if other targets ever emerge?

Alternatively we can receive a `target_triple: &str` and match:

```rust
match target_triple {
    "x86_64-pc-windows-msvc" => Ok(what you have already),
    x => Err(format!("No precompiled FidelityFX libraries available for target `{x}`")),
}
```

Reply:
I think `dll_path` implies its windows right?